### ISSUE TITLE
Category shows Passed when test Failed - version 3.0.2

### ISSUE THREAD

**Issue by dxsmith** [anshooarora/extentreports-java#777]
The Category incorrectly shows a pass when the test failed. I saw the other resolved and open issues concerning this but there doesn't appear to be a clear id of the issue yet in version 3.0.2. This may be similar to [#757](https://github.com/anshooarora/extentreports-java/issues/757). Screenshots and sample code to reproduce are shown below:

![image](https://cloud.githubusercontent.com/assets/25830708/23042489/27f9ca18-f456-11e6-8f59-050a0d64c0b9.png)

![image](https://cloud.githubusercontent.com/assets/25830708/23042272/4604b5be-f455-11e6-8989-4032c41d8bc8.png)

![image](https://cloud.githubusercontent.com/assets/25830708/23042296/5df66fe6-f455-11e6-8893-324ae07872ad.png)

```
import com.aventstack.extentreports.ExtentReports;
import com.aventstack.extentreports.ExtentTest;
import com.aventstack.extentreports.reporter.ExtentHtmlReporter;

public class TestExtent {

	public static void main(String[] args) {
		ExtentReports extent = new ExtentReports();
		ExtentHtmlReporter htmlReporter = new ExtentHtmlReporter("./extent.html");
		extent.attachReporter(htmlReporter);
		ExtentTest test;
		test = extent.createTest("Main", "Main Test desc");
		test.assignCategory("abc");
		test.pass("pass");
		test.fail("fail");
		extent.flush();
	}
}
```

**Comment by anshooarora** [comment]
Hi - will you be able to test with the latest commit?

**Comment by dxsmith** [comment]
I was able to download the code and rebuild the "extentreports-3.0.2.jar" file. I swapped out the jar and retested. The category now shows a failure. I would say that fixed it. Thanks! I appreciate your work and being so responsive!

**Comment by anshooarora** [comment]
Thanks for checking.  This fix will be part of 3.0.3.

**Comment by Vinisha-papajohns** [comment]
Hi @anshooarora,

I am still facing the issue wherein Categories are showing all the tests as Fail.
